golang/cosmos/x/swingset/keeper/keeper.go

```diff
@@ -29,6 +31,10 @@ const (
 	StoragePathBundles      = "bundles"
 )
 
+const MaxUint53 = 9007199254740991 // Number.MAX_SAFE_INTEGER = 2**53 - 1
```

Comment:
```suggestion
const MaxUint53 = 2<<52 - 1 // Number.MAX_SAFE_INTEGER = 2**53 - 1
```

Reply:
Oh I had tried something like that, and couldn't figure it out in Go

Reply:
Should fix either the code or the comment to settle on 52 vs 53.

Reply:
And the right answer: can exactly store 2**53 in a float, 2**53 + 1 is the first integer that can't be represented in a double float.

Reply:
The code and the comment are both correct: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
To clarify we can rewrite it as `1<<53 - 1` next time we touch this.